presto-tests/src/test/java/io/prestosql/tests/TestJoinQueriesWithoutDynamicFiltering.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.AbstractTestJoinQueries;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tests.tpch.TpchQueryRunnerBuilder;
+import org.testng.annotations.Test;
+
+/**
+ * @see TestJoinQueries for tests with dynamic filtering enabled
+ */
+public class TestJoinQueriesWithoutDynamicFiltering
+        extends AbstractTestJoinQueries
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return TpchQueryRunnerBuilder.builder()
+                .setExtraProperties(ImmutableMap.of("enable-dynamic-filtering", "false"))
```

Comment:
use `ENABLE_DYNAMIC_FILTERING` static import instead

Reply:
ENABLE_DYNAMIC_FILTERING is for session property.